app/models/device_supplier/user_report.rb

```diff
@@ -22,7 +22,7 @@ def add_report_rows
         .relevant_to_device_supplier
         .includes(:schools, :responsible_body, :last_user_change, schools: :responsible_body)
         .find_each do |user|
-      csv << csv_row(user)
+      csv << csv_row(user) if user.relevant_to_computacenter?
```

Comment:
Let's see how this performs in the real world and refine the scope later if necessary.

Reply:
Yes, that is the alternative but that would make that scope very complex.